packages/tools/devtools/devtools-view/package.json

```diff
@@ -50,6 +50,7 @@
 		"@fluidframework/driver-definitions": "workspace:~",
 		"@fluidframework/protocol-definitions": "^1.1.0",
 		"@fluidframework/telemetry-utils": "workspace:~",
+		"@types/chrome": "0.0.232",
```

Comment:
Is this needed for something specific? Our React view really shouldn't need to know anything about chrome's APIs if we can help it.